lib/server.js

```diff
@@ -0,0 +1,99 @@
+/*!
```

Comment:
IMO, copying over the contents of `browser.js` into a new file and extracting unused methods can eventually put us into a bad state. It'll be more difficult to maintain parity between browser and server flows.

As an example, if we decided to rename `tx.status` to `tx.get`, this would need to be done in **two** locations instead of one.

Alternatively, what do you think about importing the `browser.js` object and filter out all `token`, `session`, and `tokenManager` imports?

Reply:
I can externalize the shared code and import in `browser.js` and `server.js`, but importing directly `browser.js` I think would be confusing.

Reply:
Agreed, let's factor out the common interface to reduce duplication.